json-rpc/src/util.rs

```diff
@@ -207,3 +209,22 @@ pub fn script_view_from_script_function(script: &ScriptFunction) -> ScriptView {
         ..Default::default()
     }
 }
+
+pub fn sdk_language_from_user_agent(user_agent: Option<&str>) -> String {
```

Comment:
Is there a reason this function takes in an `Option` when it looks like its only ever called in prod code as `Some(user_ageng)`?

Reply:
Also, in order to better limit the cardinality (and avoid allocations) it might be better to have this return an Enum:

```
enum SdkLang {
    Rust,
    Go,
    Java,
    ...
}
```

And then provide an `as_str` method to convert the enum to a &str for the actual metric bump